fms2_io/netcdf_io.F90

```diff
@@ -767,6 +768,50 @@ subroutine append_compressed_dimension(fileobj, dim_name, npes_corner, &
   fileobj%compressed_dims(n)%nelems = sum(fileobj%compressed_dims(n)%npes_nelems)
 end subroutine append_compressed_dimension
 
+!> @brief Add a "compressed" unlimited dimension to a netcdf file.
+!! @note Here compressed means that every rank has a different dimension_length
+!compressed. This was written specifically for the icebergs restarts.
```

Comment:
needs another !, otherwise i think doxygen won't match up the description correctly